feature-configs/5g-ref-nw/profile-base/du-general/ptp/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- ptpconfig-slave.yaml
```

Comment:
rename for inclusivity?

Reply:
I know we are about to fix things for inclusivity. My thought here was the timing. Since we already have CI testing on the DU profile, it will take time until we upgrade the pipelines to do something meaningful tor all the building blocks we define here. In the meantime, @yuvalk  suggested that we might want to test that the new profile output is "object-exact" (not bit-exact, but produce the same objects with the same settings) as the old profile.
So what I did as a rule of thumb in this project, for everything related to the DU - keep it object-exact with the current DU profile.
This is a point I am happy to open for discussion :)

Reply:
maybe capture as a TODO